be/src/exec/broker_scanner.cpp

```diff
@@ -585,24 +601,35 @@ bool BrokerScanner::fill_dest_tuple(const Slice& line, Tuple* dest_tuple, MemPoo
             continue;
         }
 
-        ExprContext* ctx = _dest_expr_ctx[ctx_idx++];
+        ExprContext* ctx = _dest_expr_ctx[ctx_idx];
         void* value = ctx->get_value(_src_tuple_row);
         if (value == nullptr) {
-            if (slot_desc->is_nullable()) {
-                dest_tuple->set_null(slot_desc->null_indicator_offset());
-                continue;
-            } else {
+            if (_strict_mode && !_src_tuple->is_null(slot_desc->null_indicator_offset()) 
+                && !_has_expr_columns[ctx_idx++]) {
```

Comment:
_strict_mode && _has_expr_clumns[] && is_null

And don't add ctx_id++ here.

First this predicate may not be executed, and it is difficult to do things right